src/Templating/test/Templates.Test/Helpers/TemplatePackageInstaller.cs

```diff
@@ -21,7 +21,7 @@ internal static class TemplatePackageInstaller
             "Microsoft.DotNet.Web.ProjectTemplates.1.x",
             "Microsoft.DotNet.Web.ProjectTemplates.2.0",
             "Microsoft.DotNet.Web.ProjectTemplates.2.1",
-            "Microsoft.DotNet.Web.Spa.ProjectTemplates",
+            "Microsoft.DotNet.Web.Spa.ProjectTemplates.2.1",
```

Comment:
@ryanbrandenburg should this line include the `.2.1` suffix in the 'release/2.2` branch now? (Next line is `"Microsoft.DotNet.Web.Spa.ProjectTemplates.2.2"` at the moment.)

Reply:
Actually I think it should include everything. That is:
```
"Microsoft.DotNet.Web.Spa.ProjectTemplates",
"Microsoft.DotNet.Web.Spa.ProjectTemplates.2.1",
"Microsoft.DotNet.Web.Spa.ProjectTemplates.2.2",
"Microsoft.DotNet.Web.Spa.ProjectTemplates.3.0"
```
That way we know for sure that when we do `dotnet new angular` we're getting the one we just built and not just a previous or future framework.

Reply:
:ok: this is no longer a no-op merge. Have another quick look @ryanbrandenburg